webview_panels/src/modules/defer/DeferToProduction.tsx

```diff
@@ -114,6 +131,62 @@ const DeferToProduction = (): JSX.Element => {
     }
   };
 
+  const handleManifestPathTypeChange = async (option: string) => {
+    if (option === "remote") {
+      const response = await executeRequestInSync(
+        "fetchProjectIntegrations",
+        {},
+      );
+      if (Array.isArray(response)) {
+        setDeferState((prevState) => ({
+          ...prevState,
+          projectIntegrations: response.map(
+            (item: { name: string; id: number }) => ({
+              label: item.name,
+              value: item.id,
+            }),
+          ),
+          manifestPathType: option,
+        }));
+        return;
+      }
+    }
+    setDeferState((prevState) => ({
+      ...prevState,
+      manifestPathType: option,
+    }));
+  };
+
+  const handleIntegrationSelect = async (selectedOption: {
+    label: string;
+    value: number;
+  }) => {
+    const response = await executeRequestInSync("testRemoteManifest", {
+      dbt_core_integration_id: selectedOption.value,
+    });
+    const data = response as {
+      url: string;
+      dbt_core_integration_file_id: number;
+    };
+    if (data.url === "" && data.dbt_core_integration_file_id === -1) {
+      executeRequestInAsync("showInfoNotification", {
+        infoMessage: `No remote manifest file present for dbt core integration: ${selectedOption.label}`,
+      });
```

Comment:
Should we block on this? I think we should still updateDeferConfig.

Reply:
There's no reason to update defer config if there's no project integration imo